### PR TITLE
Add files via upload

### LINKED ISSUE #32: Population data

### PR DESCRIPTION
**Pull request by william-wash**
Metadata changes to support the demographics table. Closes #32

### CODE REVIEW

File: notebooks/METADATA.ipynb [hunk 1]
```diff
@@ -1,8 +1,10 @@
 {
  "cells": [
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
```
Comment (tfoldi) [inline notebooks/METADATA.ipynb]: why did you change this cell from code to markdown?

File: notebooks/METADATA.ipynb [hunk 2]
```diff
@@ -859,6 +861,94 @@
     "}]"
    ]
   },
+  {
+   "cell_type": "code",
```
Comment (tfoldi) [inline notebooks/METADATA.ipynb]: this should be markdown too

**Review by tfoldi — CHANGES_REQUESTED** [review]

File: notebooks/METADATA.ipynb (hunk 2 above)
Comment (william-wash) [reply notebooks/METADATA.ipynb]: Sorry, missed those. Fixed now.
